Makefile

```diff
@@ -1,5 +1,5 @@
 # Y - in response to Would you like to add PhraseApp configuration?
```

Comment:
@rafayet-monon could you help me to adjust this message too 🙏  in response to Would you like to add **the** PhraseApp configuration?

Reply:
Fixed in [3214e7e](https://github.com/nimblehq/rails-templates/pull/237/commits/3214e7ef3db81e6f10b8207155aa6d00ee169ef3)